openedx_events/enterprise/data.py

```diff
@@ -0,0 +1,54 @@
+"""
+Data attributes for events within the architecture subdomain `enterprise`.
+
+These attributes follow the form of attr objects specified in OEP-49 data
+pattern.
+"""
+import attr
+
+
+@attr.s(frozen=True)
+class LicenseLifecycle:
```

Comment:
What do you think of `SubscriptionLicenseData`?
1. See separate PR comments around "lifecycle".
2. I think there is a pattern of using `Data` as a suffix for these objects.